macros/src/contract.rs

```diff
@@ -113,9 +117,11 @@ pub mod interface {
 }
 
 pub mod utils {
-    use proc_macro2::{Ident, TokenStream};
-    use quote::{quote, TokenStreamExt};
-    use syn::{FnArg, Pat, TraitItemMethod, Type, TypePath};
+    use proc_macro2::{Ident, Span, TokenStream};
+    use quote::{format_ident, quote, TokenStreamExt};
+    use syn::{punctuated::Punctuated, token, FnArg, Pat, Token, TraitItemMethod, Type, TypePath};
+
+    use crate::parser::ContractTrait;
```

Comment:
I don't think it's a good idea to name a struct `ContractTrait`. Maybe rename it to `ContractDef`?

Reply:
Changed already in #36